src/Serialization/Address.purs

```diff
@@ -100,11 +107,12 @@ import Serialization.Hash (Ed25519KeyHash, ScriptHash)
 import Serialization.Types (Bip32PublicKey)
 import ToData (class ToData, toData)
 import Types.Aliases (Bech32String, Base58String)
+import Types.BigNum (BigNum)
 import Types.ByteArray (ByteArray)
 import Types.CborBytes (CborBytes)
 import Types.PlutusData (PlutusData(Bytes))
 
-newtype Slot = Slot UInt
+newtype Slot = Slot BigNum
```

Comment:
@vvtran It would be nice to deprecate `AbsSlot`, but I have some doubts regarding that small max value difference between `BigNum` and Ogmios Integer (see my comment on the max value).

Reply:
@errfrom  I replied to the other post